apps/api/tests/test_sso_okta.py

```
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+from uuid import UUID
+
+from sqlmodel import Session
+
+from app.database import get_engine
+from app.models import Organization
+
+
+def _register(client, *, email: str, password: str = "Password123!", organization_name: str | None = None) -> dict:
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to function default: "password"

(S107)